commit 80865f7579d619dee4ffd5abb72f641ef74dedbb
Author: Wei Mi <wmi@google.com>
Date:   Fri Sep 24 15:35:07 2021 -0700

    Add "REQUIRES: zlib" in forward-compatible.test since it handles compressed file.
---
 llvm/test/tools/llvm-profdata/forward-compatible.test | 1 +
 1 file changed, 1 insertion(+)

diff --git a/llvm/test/tools/llvm-profdata/forward-compatible.test b/llvm/test/tools/llvm-profdata/forward-compatible.test
index 8d45ba7d8a73..51a7336bb405 100644
--- a/llvm/test/tools/llvm-profdata/forward-compatible.test
+++ b/llvm/test/tools/llvm-profdata/forward-compatible.test
@@ -1,3 +1,4 @@
+REQUIRES: zlib
 RUN: llvm-profdata show -sample %S/Inputs/unknown.section.extbin.profdata | FileCheck %s
 RUN: llvm-profdata show -sample %S/Inputs/unknown.section.compressed.extbin.profdata | FileCheck %s
 RUN: llvm-profdata show -sample -show-sec-info-only %S/Inputs/unknown.section.extbin.profdata | FileCheck %s -check-prefix=HDR
